package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "grep-components",
-  "version": "0.6.0",
+  "version": "0.6.1-gref-910-dev.0",
```

Comment:
Blir dette riktig versjonsnummer på dev @odinr ?

Reply:
ja, for det er eb prerealse, etter patch blir den 6.1